code/registration.py

```diff
@@ -269,6 +263,171 @@ def load_initial_frames(
     return frames
 
 
+def combine_images_with_individual_titles(image1_path, image2_path, output_path, title1="", title2=""):
```

Comment:
Can you add types to the inputs? E.g

`def combine_images_with_individual_titles(image1_path: str, image2_path: str, output_path: str, title1: str =1, title2: str=2`

Reply:
Added parameter types